### DISCUSSION TITLE
Upgrade base Dockerfille.helm3 with one of the helm minor version (3.8.0/3.9.0)

### DISCUSSION THREAD

**Discussion by Jasstkn** [helmfile/helmfile#160] [Questions] [answered]
Hey.

I would like to use helmfile base image with one of the latest version of helm:
either 3.8.0 or 3.9.0

Is there any objections from your side to submit PR with these changes?

https://github.com/helmfile/helmfile/pull/163

**Comment by yxxhero** [comment, marked as answer]
Welcome for the PR.

**Reply by Jasstkn** [reply]
Do you know why images build is being skipped?
